feat(UI): Histogram

yaelsuarez opened the pull request:
Fixes #48 and fixes #49

Commit:
feat(UI): Favico is now the logo

yaelsuarez committed:
Merge branch 'fcossio:main' into main

yaelsuarez committed:
Merge branch 'fcossio:main' into main

yaelsuarez committed:
Merge branch 'fcossio:main' into main

yaelsuarez committed:
Merge branch 'fcossio:main' into main

yaelsuarez committed:
feat(UI): Histogram

yaelsuarez commented:
Hej hej @coszio. We added the histogram feature, but we were not able to make it have a fixed position on the bottom of the side panel above Globals section, maybe you have some suggestions for this? Any other suggestions on style, size, etc are welcome as well :) Tack!

coszio commented:
This is great work! I didn't know about Victory Charts. 

If you don't mind I can make these changes myself. I'd like to:
- use a debounced value at the input data of the histogram, to avoid excessive rerenders
- fit the `Histogram` section to the bottom, next to `Globals`
- add labels to the axes to make it more clear.
- if possible add some text when there is no data, something like 
  > Add some lines to begin plotting

coszio committed:
fix(UI): improve histogram section

- use a debounced value at the input data of the histogram, to avoid excessive rerenders
- fit the Histogram section to the bottom, next to Globals
- add labels to the axes to make it more clear.
- add text label when there is no data

coszio commented:
Done! @yaelsuarez please review these changes. If there is something else to make it better, we can do that before merging 😄. If not, I'd say it is safe to proceed to `Squash and merge` these changes.

yaelsuarez committed:
fix(UI): Updated test image

yaelsuarez committed:
fix(UI): fixed number of bins on the histogram

yaelsuarez committed:
fix(UI): Decreased time of
"No fiber found" toast
